testing/mysql/APKBUILD

```
@@ -0,0 +1,299 @@
+# Maintainer:
+pkgname=mysql
+pkgver=8.0.16
```

```suggestion
pkgver=8.0.17
```

Also update the sums.